provider/ec2/environ.go

```diff
@@ -12,6 +12,7 @@ import (
 	"sync"
 	"time"
 
+	sdkec2 "github.com/aws/aws-sdk-go/service/ec2"
```

Comment:
how about leave this unaliased and alias "gopkg.in/amz.v3/ec2" as legacyec2 ?

Reply:
Sounds good, though I prefer something like `amzec2` as it's more specific.